### PR TITLE
Added note for static files not checked - git submodule error

### PR DESCRIPTION
**Pull request by geekashu**
Added note [#709](https://github.com/yourlabs/django-autocomplete-light/issues/709)

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -23,6 +23,10 @@ This is to override the ``jquery.init.js`` script provided by the admin, which
 sets up jQuery with ``noConflict``, making jQuery available in
 ``django.jQuery`` only and not ``$``.
 
+Note: If you are trying to install from git, please make sure you are not using **zip/archive** url of \ 
+the repo ``django-autocomplete-light`` since it will not contain required submodules automatically. \
```
Comment (jpic) [inline docs/install.rst]: Could you remove the trailing `\` and word wrap to 80 caracters ? (if using vim, select the paragraph in visual mode and type gqq)

**Comment by geekashu** [comment]
@jpic Thank you for encouraging me to submit this. This is my first contribution... :)

**Comment by jpic** [comment]
Looks perfect !

Could you just squash the commits into one ? You can do `git rebase -i origin/master`, replace the two last "pick" with "fixup", and `git push -f` your branch.

Thankssss !!

**Comment by jpic** [comment]
If you can't it's ok we can merge it as is, it's just that it would be slightly better to have this squashed into one commit, let me know if you can ;)

**Comment by jpic** [comment]
> @jpic Thank you for encouraging me to submit this. This is my first contribution... :)

And you're very welcome any time, we're all here to learn to make better open source software to build a free internet as a free communication tool for everybody on the network (i would say "for everybody in the world" but there's China).

**Comment by geekashu** [comment]
Done... Thanks

**Comment by jpic** [comment]
Mind blowing, you :guitar:

**Pull request merged**